plugin.php

```diff
@@ -10,12 +10,14 @@
 namespace Altis\Experiments;
 
 const ROOT_DIR = __DIR__;
+const ROOT_FILE = __FILE__;
 
 // Check if this is installed as a self contained built version.
 if ( file_exists( ROOT_DIR . '/vendor/autoload.php' ) ) {
 	require_once ROOT_DIR . '/vendor/autoload.php';
 }
 
 require_once 'inc/namespace.php';
+require_once 'inc/utils/namespace.php';
```

Comment:
These should be full paths so we're not relying on include_path

```suggestion
require_once ROOT_DIR . '/inc/namespace.php';
require_once ROOT_DIR . '/inc/utils/namespace.php';
```